packages/client/components/ActivityLibrary/ActivityDetails.tsx

```diff
@@ -156,115 +158,166 @@ const ActivityDetails = (props: Props) => {
-                          >
-                            <ContentCopy className='text-slate-600' />
-                            <div className='font-semibold text-slate-700'>Clone & Edit</div>
-                          </FlatButton>
-                        </div>
+                      <b>Reflect</b> on what’s working or not on your team. <b>Group</b> common
+                      themes and vote on the hottest topics. As you <b>discuss topics</b>, create{' '}
+                      <b>takeaway tasks</b> that can be integrated with your backlog.
+                    </div>
+                    <div className='mt-[18px] flex min-w-max items-center'>
+                      <div className='flex items-center gap-3'>
+                        <JiraSVG />
+                        <GitHubSVG />
+                        <JiraServerSVG />
+                        <GitLabSVG />
+                        <AzureDevOpsSVG />
                       </div>
-                    )}
-                  </div>
-                  <b>Reflect</b> on what’s working or not on your team. <b>Group</b> common themes
-                  and vote on the hottest topics. As you <b>discuss topics</b>, create{' '}
-                  <b>takeaway tasks</b> that can be integrated with your backlog.
-                </div>
-                <div className='mt-[18px] flex min-w-max items-center'>
-                  <div className='flex items-center gap-3'>
-                    <JiraSVG />
-                    <GitHubSVG />
-                    <JiraServerSVG />
-                    <GitLabSVG />
-                    <AzureDevOpsSVG />
-                  </div>
-                  <div className='ml-4'>
-                    <b>Tip:</b> push takeaway tasks to your backlog
+                      <div className='ml-4'>
+                        <b>Tip:</b> push takeaway tasks to your backlog
+                      </div>
+                    </div>
                   </div>
+                  <TemplatePromptList
+                    isOwner={isOwner && isEditing}
+                    prompts={prompts!}
+                    templateId={templateId}
+                  />
+                  {isOwner && isEditing && (
+                    <AddTemplatePrompt templateId={templateId} prompts={prompts!} />
+                  )}
                 </div>
               </div>
-              <TemplatePromptList isOwner={isOwner} prompts={prompts!} templateId={templateId} />
-              {isOwner && <AddTemplatePrompt templateId={templateId} prompts={prompts!} />}
             </div>
           </div>
+          <ActivityDetailsSidebar
+            selectedTemplateRef={selectedTemplate}
+            teamsRef={teams}
+            isOpen={!isEditing}
+          />
         </div>
-        <ActivityDetailsSidebar selectedTemplateRef={selectedTemplate} teamsRef={teams} />
+        {isEditing && (
+          <div className='absolute bottom-0 flex h-20 w-full items-center justify-center bg-slate-200'>
```

Comment:
-1 there's no explicit `relative` element in this view so we shouldn't be using `absolute` and `bottom-0` here. this element sits in `flex` container so it's better to just use `mt-auto` without `absolute` and `bottom-0`

Reply:
This is intentional so that this element sits above the sidebar during the animation. If this is positioned normally, the sidebar will jump a bit as this element displaces it.

Reply:
ah I see. what about setting root div to have positon `relative`? without that element will be implicititly positioned to nearest element with position `relative` or root (body). it'll work until someone adds a div that'll be somewhere in the hierarchy and could break this view without knowing

Reply:
Ah gotcha. Added `relative` to the root div in this component